language/examples/bqml/bqml_llm_examples.ipynb

```diff
@@ -0,0 +1,1131 @@
+{
```

Comment:
<p>Header here should also be "Using LLMs with BigQuery ML" perhaps</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>updated - thanks!</p>